src/components/calculator.js

```diff
@@ -1,55 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-  }
+const Calculator = () => {
+  // constructor(props) {
+  //   super(props);
+  //   this.state = {
+  //     total: null,
+  //     next: null,
+  //     operation: null,
+  //   };
+  // }
```

Comment:
- [ ] [OPTIONAL] Kindly remove the comment code from your code as it already exists in the commit history in order to make your code more professional.

Reply:
Thanks @gemmen29